packages/shared_preferences/shared_preferences_windows/example/integration_test/shared_preferences_test.dart

```diff
@@ -30,8 +30,8 @@ void main() {
 
     late SharedPreferencesWindows preferences;
 
-    setUp(() async {
-      preferences = SharedPreferencesWindows.instance;
+    setUpAll(() async {
```

Comment:
This makes the object under test global for all tests, which is exactly what we don't want. At the least this should be reverted; preferably `late SharedPreferencesWindows preferences;` should be removed entirely and the test object created in each test.